docs/app-integration-development/06-Events-Integration.md

```diff
@@ -102,21 +102,17 @@ Use an Event Transformer when a technical service you are connecting to PagerDut
 
 2. If you haven't yet, create a test service (see [Test your integration](#test-your-integration) below)
 
-3. Turn on **Debug Mode** just below the code editor to trigger an incident on your test service when a runtime error occurs during your testing. This allows you to debug your transformer code.
-
-![Screenshot of dropdown to enable Event Transformer debug mode](../../assets/images/enable-debug-mode.png)
-
-4. Click **Save** at the bottom of the page to deploy the default transform.
+3. Click **Save** at the bottom of the page to deploy the default transform.
```

Comment:
I think you need them to save first, then link out to testing instructions.

Reply:
switched: https://github.com/PagerDuty/developer-docs/pull/93/commits/e871af8d5582590d1e4b443116869c4cbe624f48#diff-5daad561c84cfde83ecdf2cbbc63de220748e7b3bd867c95737bc5731882209aR105